src/pages/releaseNote/releaseNote.js

```diff
@@ -0,0 +1,368 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import header from '../../assets/releaseNotes/headerImage.svg';
+import { defaultVersion, releaseNotePageData, versionList } from '../../bento/releaseNotePageData';
+import { ArrowDropUp, ArrowRight } from '@material-ui/icons';
+import exportIcon from '../../assets/about/Export_Icon.svg';
+
+
+const ReleaseNoteContainer = styled.div`
+ margin: 0 auto;
+ display: flex;
+ flex-direction: column;
+ aling-items: center;
+ justify-content: center;
+ .releaseNoteHeader {
+   font-family: Poppins;
+   font-weight: 500;
+   font-size: 35px;
+   background-image: url(${header});
+   line-height: 45px;
+   height: 167px;
+   letter-spacing: 0.15px;
+   color: #FFFFFF;
+   text-align: center;
+   margin-top: 0px;
+  
+   background-repeat: no-repeat;
+   background-position: center;
+   background-size: 100%;
+   background-fit: fit;
+   border-radius: 0px;
+   border: none;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   padding: 0px;
+   }
+ .releaseNoteBody {
+   display: flex;
+   justify-content: center;
+   gap: 30px;
+   margin-top: 40px;
+   background-size: cover;  /* Ensures the background image covers the entire container */
+   background-repeat: no-repeat;  /* Disable repeating */
+   background-position: center;  /* Center the image */
+   flex-direction: row;
+   align-items: flex-start;
+   background-width: 100%;
+   padding-bottom: 100px;
+   height: 800px;
+ }
+.versionSelector {
+   width: 303px;
+   height: 314px;
+   border: 2px #4D7874 solid;
+   border-radius: 20px;
+   background-color: white;  
+   overflow:hidden;
+   overflow-y: scroll;
```

Comment:
~~overflow-y: scroll;~~
overflow-y: auto;

Reply:
Any reason for this update request? scroll handled the desired outcome

Reply:
resolved